controllers/discovery/initDiscoveryTopicData.js

```diff
@@ -42,11 +51,22 @@ const InitDiscoveryTopicData = async (req, res) => {
     });
     let suggestedTopics = topicWithCommonFollowerResult;
 
+    let totalData = await sequelize.query(totalDataQuery, {
+      type: QueryTypes.SELECT,
+      replacements: {
+        userId
+      },
+      raw: true
+    });
+    totalData = totalData?.[0]?.total_data || 0;
+    console.log('totalData', totalData);
+
     return res.status(200).json({
       success: true,
       message: `Fetch discovery data success`,
       suggestedTopics,
-      page: page + 1
+      page: page + 1,
+      total_page: totalData > 0 ? Math.ceil(totalData / limit) : 0
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Logging sensitive data like `totalData` can pose security risks and should be avoided in production code. Also, error handling is missing for the case when the database query fails. Consider adding a try-catch block around the database query and handle any potential errors.
```diff
-     console.log('totalData', totalData);
+     // console.log('totalData', totalData); // remove this line

      return res.status(200).json({
        success: true,
        message: `Fetch discovery data success`,
        suggestedTopics,
       page: page + 1,
       total_page: totalData > 0 ? Math.ceil(totalData / limit) : 0
     });
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->